9-Flexbox/Part-2/css/style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  background-color: #e2e7ec;
+}
+
+.wrapper {
+  max-width: 1600px;
+}
+
+.container-fluid {
+  width: 1120px;
```

Comment:
we shouldn't rewrite that class, with that changes we have horizontal scroll and it is incorrect